README.md

```diff
@@ -24,6 +24,14 @@ For example, use conduit to:
 * Send blockchain data to another streaming data platform for additional processing (e.g. RabbitMQ, Kafka, ZeroMQ).
 * Build an NFT catalog based on different standards.
 
+# System Requirements
+
+For a simple deployment the following configuration works well:
+* Network: Conduit colocated with Algod follower.
+* Conduit: 4 CPU and 8 GB of ram.
+* Algod: 4 CPU and 8 GB of ram.
```

Comment:
Perhaps `Conduit + Algod: 4 CPU and 8 GB of RAM`? Otherwise it implies needing to dedicate that much to each process.